libs/design-system-components/src/components/NotificationSettingsCard/index.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+
+import Card from '@akashaorg/design-system-core/lib/components/Card';
+import Stack from '@akashaorg/design-system-core/lib/components/Stack';
+import Button from '@akashaorg/design-system-core/lib/components/Button';
+import Text from '@akashaorg/design-system-core/lib/components/Text';
+import Image from '@akashaorg/design-system-core/lib/components/Image';
+
+export type NotificationSettingsCardProps = {
+  // data
+  title: string;
+  text: string;
+  imageSrc?: string;
+  isLoading?: boolean;
+  showButton?: boolean;
+  // handlers
+  handleButtonClick: () => void;
+};
+
+const NotificationSettingsCard: React.FC<NotificationSettingsCardProps> = ({
+  title,
+  text,
+  imageSrc,
+  isLoading,
+  showButton = true,
+  handleButtonClick,
+}) => {
+  return (
+    <Card radius={16} customStyle="p-0 w-full grow flex-wrap">
+      <Stack padding="p-5" align="center">
+        {imageSrc ? (
+          <Image
+            customStyle="w-[180px] h-[180px] object-contain mb-4"
+            src={imageSrc}
+            alt="Notification illustration"
+          />
+        ) : (
+          <div aria-hidden="true" className="w-[180px] h-[180px] bg-[#C4C4C4] mb-4"></div>
```

Comment:
We have predefined colors https://github.com/AKASHAorg/akasha-core/blob/next/libs/typings/src/ui/design-system.ts. Is #C4C4C4 what you saw on the design system for this card ?

Reply:
Yes, #C4C4C4 is in the Figma, but i didn't see variable defined anywhere. Should I  define a new one or do we allow such excemptions?

Reply:
I see. If its a new variation of grey can you define it in the variables ?

Reply:
Ok I asked Sara, I now understand the situation, so this div is a placeholder till she finds a proper image on the design it was not meant to be created as a div. For such cases you can just leave a comment and when she gets the proper image you can replace it with a relevant component.